config/webpack.config.prod.js

```diff
@@ -98,23 +95,6 @@ module.exports = {
     return [autoprefixer];
   },
   plugins: [
-    new HtmlWebpackPlugin({
```

Comment:
you can also combine HtmlWebpackPlugin and SSR, it is limiting, e.g. doesn't support streaming (since it's not rendered by react fully), but you can totally get access to generated HTML on the server